views/github/runner.erb

```diff
@@ -68,12 +68,20 @@
           <% end %>
         <% else %>
           <tr>
-            <td colspan="5">
-              <div class="text-center p-4">
-                No active runners. In your workflow file, change the
-                <span class="text-rose-500 bg-slate-100 py-0.5 px-2 rounded font-mono whitespace-nowrap">runs-on: ...</span>
-                line to
-                <span class="text-rose-500 bg-slate-100 py-0.5 px-2 rounded font-mono whitespace-nowrap">runs-on: ubicloud</span>. Then, trigger your workflow to start a runner.
+            <td colspan="6">
+              <div class="text-center py-4 px-8 lg:px-32">
+                <h3 class="text-xl leading-10 font-medium mb-2">No active runners</h3>
+                <p class="leading-6">
+                  To use Ubicloud runners, you need to change
+                  <span class="text-rose-500 text-sm font-medium bg-slate-100 py-0.5 px-2 rounded font-mono whitespace-nowrap">runs-on: ...</span>
+                  line to one of our labels such as
+                  <span class="text-rose-500 text-sm font-medium bg-slate-100 py-0.5 px-2 rounded font-mono whitespace-nowrap">runs-on: ubicloud-standard-2</span>
+                  in your workflow file. Then, trigger your workflow to start a runner.
+                  <br>
+                  Check out
+                  <a href="https://www.ubicloud.com/docs/github-actions-integration/quickstart" class="text-orange-500 font-medium">our documentation</a>
+                  for using larger runners, using arm64, enabling GPU.
+                </p>
```

Comment:
should we move this outside of the tabs? So, we can remove the marked part:
<img width="1511" alt="371629827-4aed16e0-2049-4b41-a2df-f5dec669dac5" src="https://github.com/user-attachments/assets/6470bc32-1102-4cfe-85d1-158c26917d74">

Reply:
Actually, thinking about this more, considering we have the "No active runners" message, I think keeping this is the right way.